docs/tutorials/data-loading/databases.md

```diff
@@ -25,11 +25,15 @@ You need the following software installed:
 
 To run the example code, clone the
 [qlik-oss/core-grpc-jdbc-connector](https://github.com/qlik-oss/core-grpc-jdbc-connector)
-Git repository. Check out the repo documentation to get familiar with the content and structure.
+Git repository. Before you continue, look at the documentation to get familiar with the content and structure of
+the repository.
 
-In your shell, make sure current directory is at the repo root.
+!!!Tip
+    In your shell, make sure current directory is at the repository root.
 
-Set the `ACCEPT_EULA` environment variable as applicable and start the containers by running:
+To begin, you must accept the `ACCEPT_EULA` and start the containers.
```

Comment:
You don't really accept the `ACCEPT_EULA`. It's an env variable. Maybe just:
_To begin, you must accept the EULA and start the containers._